### DISCUSSION TITLE
"go to turbowarp" button

### DISCUSSION THREAD

**Discussion by piw-piw** [ScratchAddons/ScratchAddons#2126] [Ideas] [answered]
Add a "go to turbowarp" button on the project page.
This button detects if there is a link of this project on turbowarp in the description and provides it, or if there is no link - 
creates it without customization.

**Comment by ghost** [comment]
its imposible

**Reply by retronbv** [reply]
It is completely possible, I have a Userscript that does this (w/o parsing the description, it just always shows). I have tried to get it merged as an addon but it didn’t get merged.

**Comment by cobaltt7** [comment]
No? It's completely possible...
________________________________

**Comment by qbjl** [comment]
Yes! I can see this being useful!

A lot of projects now are being played in TurboWarp for the FPS bypass, Scratch to JS, e.t.c. I know this wouldn't be too hard, because all you would have to do is redirect to TurboWarp with the project ID.

(Side note, I think it would be useful if there were ratios on the settings page for this to toggle for settings, such as ?fps, ?interpolate, and ?hqpen)

**Comment by ghost** [comment]
ok, this is possible, its kinda useless, is better just downloading the project and import it to turbowarp! also turbowarp have similar scratch addons feautres (or the same)

**Reply by retronbv** [reply]
> ok, this is possible

we never said that it was impossible

> is better just downloading the project and import it to turbowarp!

this would be for ease of access, downloading and uploading takes too long

**Comment by cobaltt7** [comment]
You don't need to download it anyway, just copy and paste the link into TW
________________________________

**Reply by Endermaster3690** [reply]
exactly. Just add a button by the other buttons in a project that says Turbowarp and if you click it it takes you to the turbowarp page.

**Comment by Hans5958** [comment, marked as answer]
For the note: This has been implemented as `turbowarp-player`.
